lib/android.js

```diff
@@ -78,7 +88,7 @@ function getApi () {
   pending.forEach(function (api) {
     const functions = api.functions || {};
     const variables = api.variables || {};
-    const optionals = api.optionals || {};
+    const optionals = api.optionals || [];
```

Comment:
Or actually let's just make `optionals` a plain array, and do the `Set` wrapping here:

```js
    const optionals = new Set(api.optionals || []);
```